### PR TITLE
oreproc changes

### PR DESCRIPTION
**Pull request by Rundas01**

### CODE REVIEW

File: common/src/main/java/muramasa/gregtech/data/Guis.java [hunk 1]
```diff
@@ -121,7 +121,10 @@ public static void init(Side side) {
         LASER_ENGRAVER.add(ALLOY_SMELTER);
         FORMING_PRESS.add(ALLOY_SMELTER);
         FORGE_HAMMER.add(FURNACE);
-        SIFTER.add(DISASSEMBLER);
+        SIFTER.add(IT_IN, 53, 25)
+                .add(IT_OUT, 107, 16).add(IT_OUT, 125, 16).add(IT_OUT, 143, 16)
+                .add(IT_OUT, 107, 34).add(IT_OUT, 125, 34).add(IT_OUT, 143, 34)
+                .add(ENERGY, 80, 63);
```
Comment (Trinsdar) [inline common/src/main/java/muramasa/gregtech/data/Guis.java:127]: doesn't the sifter in gt5u have 9 slots? why this change?

**Review by Trinsdar — APPROVED** [review]

File: common/src/main/java/muramasa/gregtech/data/Guis.java (hunk 1 above)
Comment (Rundas01) [reply common/src/main/java/muramasa/gregtech/data/Guis.java:127]: sifter had 9 slots, although there's no recipe which requires all of them. which makes 3 of them obsolete.
